### ISSUE TITLE
Feature Request: Search inside HTML panels

### ISSUE THREAD

**Issue by lordfeck** [allinurl/goaccess#2731]
It would be convenient to use a search term to filter the contents of the html panels.

E.g my 'requested files' panel has 20+ pages of data and I only wish to see the hits on a particular page. I'd prefer to just type part of the HTTP path rather than sift through each of the 20 pages manually.

![image](https://github.com/user-attachments/assets/28908dac-283d-4166-baa1-568a39da63af)

What I have in mind is a search field somewhere here ^

Presumably because we're aldready using JavaScript and JSON for the data, this shouldn't be a big task, and it would bring some more parity with the curses UI.

**Comment by allinurl** [comment]
Does it seem like #117 would accomplish this? Working on it as we speak.

**Comment by lordfeck** [comment]
@allinurl yes it does - if basic filtering by pathname will be possible as part of that ticket.
